make_ocr_features/make_OCR_df_and_preprocess.ipynb

```diff
@@ -0,0 +1,1131 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ratio : 0.14662308272567262\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"takes approx 1min30\"\"\"\n",
+    "intersection_list_lang = parallel_calc(get_intersect_words_lang, lang_dict_list)\n",
+    "print(\"ratio:\", len([item for item in intersection_list_lang if item != \"\"]) / len(intersection_list_lang))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ratio: 0.4644321515380102\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"takes approx 56sec\"\"\"\n",
+    "sub_ocr_text_dict_list = [ocr_text_dict[code] for code in df[\"code\"]]\n",
+    "intersection_list_ocr = parallel_calc(get_intersect_words_ocr, sub_ocr_text_dict_list)\n",
+    "print(\"ratio:\", len([item for item in intersection_list_ocr if item != \"\"]) / len(intersection_list_ocr))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df[\"intersection_words_lang\"] = intersection_list_lang\n",
+    "df[\"intersection_words_ocr\"] = intersection_list_ocr"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "315604    30\n",
+       "70086     30\n",
+       "32122     30\n",
+       "275024    30\n",
+       "115303    30\n",
+       "          ..\n",
+       "261679     0\n",
+       "261676     0\n",
+       "261675     0\n",
+       "261672     0\n",
+       "596549     0\n",
+       "Name: intersection_words_ocr, Length: 596550, dtype: int64"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df[\"intersection_words_ocr\"].str.split().str.len().sort_values(ascending = False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fetch big words from images"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import gzip\n",
+    "import json\n",
+    "from utils.OCR_word_selection import get_big_words_from_txt_annotations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "big_words_dict = {}\n",
+    "i_last = -1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0\n",
+      "250000\n",
+      "500000\n",
+      "750000\n",
+      "1000000\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"takes approx 7m30\"\"\"\n",
+    "txt_annotations_path = \"INPUT_datasets/txtannotations.jsonl.gzip\"\n",
```

Comment:
Where does that file come from? -> add to README
(I couldn't run notebook further, will do when I have the file)